ansible/roles/hoover/tasks/main.yml

```diff
@@ -94,20 +94,37 @@
     remote_src: yes
     creates: /opt/hoover/elasticsearch-2.4.3
 
+- name: Set up file permissions for elasticsearch
+  file:
+    path: /opt/hoover/elasticsearch-2.4.3
+    owner: liquid-apps
+    group: liquid-apps
+    state: directory
+    recurse: true
+
 - name: Write the elasticsearch configuration file
   template:
     src: elasticsearch-config/elasticsearch.yml
     dest: /opt/hoover/elasticsearch-2.4.3/config/elasticsearch.yml
   tags:
     - configure
 
-- name: Change ownership of elasticsearch data and logs folders
+- name: Create hoover data and log dirs under /var/lib/liquid/hoover
   file:
-    path: "/opt/hoover/elasticsearch-2.4.3"
+    path: "/var/lib/liquid/{{item}}"
     owner: liquid-apps
     group: liquid-apps
     state: directory
-    recurse: true
```

Comment:
No more recursive, why?

Reply:
this does something else now; that segment of code moved to line 96